gravitee-apim-console-webui/src/management/api/creation-v4/steps/step-3-endpoints/step-3-endpoints-1-list.component.html

```diff
@@ -61,6 +61,19 @@
       </mat-list-option>
     </mat-selection-list>
 
+    <div *ngIf="shouldUpgrade$ | async">
+      <gio-banner type="error" icon="gio:universe" class="license__banner">
```

Comment:
As we repeat it a lot of time do you think it can be interesting to create a dedicated small component ? It would allow use to update the message one time if in the future we want to change something. I would keep if like that if we have different message from a page to another